src/hooks/use-arrow-key-navigation.ts

```diff
@@ -92,7 +91,7 @@ export function useArrowKeyNavigation(
   // Keep track of the element that was last focused by this hook such that
   // navigation can be restored if focus moves outside the container and then
   // back to/into it.
-  const lastFocusedItem = useRef<HTMLOrSVGElement | null>(null);
+  const lastFocusedItem = useRef<HTMLElement | null>(null);
```

Comment:
Is this because `getNavigableElements` returns `HTMLElement[]`?

Reply:
In essence yes, but I could not find a reason for the ref to be typed as `HTMLOrSVGElement`.

I'm ok doing an inline `as HTMLElement` cast though. We are already doing it in a couple of other places for this hook.